snippets/xctest.code-snippets

```diff
@@ -0,0 +1,22 @@
+{
+  "XCTestSuite": {
+    "prefix": "testsuite",
+    "body": [
+      "class ${1:suiteName}: XCTestCase {",
+      "\tfunc test_${2:testName}() {",
+      "\t\t$2",
+      "\t}",
+      "}"
+    ],
+    "description": "XCTest: Test Suite"
+  },
+  "XCTest": {
+    "prefix": "test",
+    "body": [
+      "func test${1:Name}() {",
+      "\t$2",
+      "}"
+    ],
+    "description": "XCTest: Test Function"
```

Comment:
```suggestion
    "description": "XCTest: Test Method"
```
"method" is generally the preferred term in the context of XCTest, since it only supports tests as methods. In contrast, swift-testing supports global functions so we've been using "function" there.